src/pages/ProfilePage.tsx

```diff
@@ -219,30 +204,30 @@ function ProfilePage(props) {
                                 <MenuItemWithTopDescription
                                     shouldShowRightIcon
                                     title={notificationPreference}
-                                    description={props.translate('notificationPreferencesPage.label')}
-                                    onPress={() => Navigation.navigate(ROUTES.REPORT_SETTINGS_NOTIFICATION_PREFERENCES.getRoute(props.report.reportID))}
+                                    description={translate('notificationPreferencesPage.label')}
+                                    onPress={() => Navigation.navigate(ROUTES.REPORT_SETTINGS_NOTIFICATION_PREFERENCES.getRoute(report.reportID))}
                                     wrapperStyle={[styles.mtn6, styles.mb5]}
                                 />
                             )}
-                            {!isCurrentUser && !Session.isAnonymousUser() && (
+                            {!isCurrentUser && !SessionActions.isAnonymousUser() && (
                                 <MenuItem
-                                    title={`${props.translate('common.message')}${displayName}`}
+                                    title={`${translate('common.message')}${displayName}`}
                                     titleStyle={styles.flex1}
                                     icon={Expensicons.ChatBubble}
-                                    onPress={() => Report.navigateToAndOpenReportWithAccountIDs([accountID])}
+                                    onPress={() => ReportActions.navigateToAndOpenReportWithAccountIDs([accountID])}
                                     wrapperStyle={styles.breakAll}
                                     shouldShowRightIcon
                                 />
                             )}
-                            {!_.isEmpty(props.report) && !isCurrentUser && (
+                            {!isEmptyObject(report) && !isCurrentUser && (
                                 <MenuItem
-                                    title={`${props.translate('privateNotes.title')}`}
+                                    title={`${translate('privateNotes.title')}`}
                                     titleStyle={styles.flex1}
                                     icon={Expensicons.Pencil}
-                                    onPress={() => ReportUtils.navigateToPrivateNotes(props.report, props.session)}
+                                    onPress={() => ReportUtils.navigateToPrivateNotes(report, session)}
                                     wrapperStyle={styles.breakAll}
                                     shouldShowRightIcon
-                                    brickRoadIndicator={Report.hasErrorInPrivateNotes(props.report) ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : ''}
+                                    brickRoadIndicator={ReportActions.hasErrorInPrivateNotes(report) ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined}
```

Comment:
Should we fall back to empty string?
Prop definition:
```js
/** The type of brick road indicator to show. */
brickRoadIndicator: PropTypes.oneOf([CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR, CONST.BRICK_ROAD_INDICATOR_STATUS.INFO, ''])
```

Reply:
Nope, with TS updates we can have one of the `CONST.BRICK_ROAD_INDICATOR_STATUS` values or undefined